### DISCUSSION TITLE
Problem in running

### DISCUSSION THREAD

**Discussion by PouyaFakhari** [lllyasviel/Fooocus#2914] [Q&A] [answered]
Hello to all , I waited for the installation on my system and finally after downloading the window opened and now I get this error every time it doesn't do anything. Where is the problem?

![Screenshot (991)](https://github.com/lllyasviel/Fooocus/assets/135238006/f8a99ecd-5131-4b14-b51e-39ff3be278f0)
![Screenshot (990)](https://github.com/lllyasviel/Fooocus/assets/135238006/42d2805e-e91e-4387-90ad-9872cd932a15)

**Comment by PouyaFakhari** [comment, marked as answer]
Update: After trying again to turn off the internet and use it offline, the problem was solved, thank you

**Reply by 15xa** [reply]
let me try it > crying since an 4 hrs from same issue

**Reply by 15xa** [reply]
nope, still the same 
![image](https://github.com/lllyasviel/Fooocus/assets/62839670/3f5e848a-b374-4e30-a618-51c107ac78f5)

**Reply by 15xa** [reply]
i don't even understand why  mine is white and the whole planet got it dark

**Reply by PouyaFakhari** [reply]
im kinda new but i think its because of your internet or models does not installed corectly (my problem fix it by just internet now when my internet connected or not it work ok) try use v.p.n or something or reinstall im not offical helper im kinda new but i hope it help

**Reply by mashb1t** [reply]
@adityap1515 you're not using dark mode, that's why ;)
EDIT: you can also set the theme with an argument, see readme.
Please check the terminal (cmd) output and find any errors there. If you can't solve the issue feel free to open another discussion or an issue.
